src/components/AssetHeader/AssetActions.tsx

```diff
@@ -1,11 +1,16 @@
 import { ArrowDownIcon, ArrowUpIcon } from '@chakra-ui/icons'
 import { Button, ButtonGroup, Skeleton } from '@chakra-ui/react'
+import { Asset, MarketData } from '@shapeshiftoss/types'
 import { useModal } from 'context/ModalProvider/ModalProvider'
 import { useWallet, WalletActions } from 'context/WalletProvider/WalletProvider'
-import { useAsset } from 'pages/Assets/Asset'
 
-export const AssetActions = ({ isLoaded }: { isLoaded: boolean }) => {
-  const { asset, marketData } = useAsset()
+type AssetActionProps = {
```

Comment:
to reduce renders, we should be passing in just the `assetId` here, and having this component get the asset data and marketData with selectors